### ISSUE TITLE
Cannot close the shield when VO is on

### ISSUE THREAD

**Issue by JuliannaR** [cds-snc/covid-shield-mobile#186]
**Scenario to reproduce**

Then the shield is open you cannot double tap on the arrow in the right corner to close, no gesture allows you to change screens. Tapping near the arrow opens the how it works screens.

**Expected result**

To be able to close the shield and return to the content underneath.

**Bug or issue / WCAG**

2.5.1: Pointer Gestures
[https://www.w3.org/WAI/WCAG21/Understanding/pointer-gestures.html
](url)

**Screenshots where available**

![Image from iOS](https://user-images.githubusercontent.com/35537308/85855707-b183ea80-b784-11ea-8ab3-a1e280271fff.png)

**Device + OS**

iPhone 8 | 13.5.1

**Comment by JuliannaR** [comment]
Cross linked to https://github.com/CovidShield/mobile/issues/169

**Comment by JuliannaR** [comment]
Cross linked to https://github.com/CovidShield/mobile/issues/170

**Comment by JuliannaR** [comment]
Merged into this issue: https://github.com/cds-snc/covid-shield-mobile/issues/432
